try:
    print(x)
except:
    print("this error")   